spring-cloud-dataflow-server-core/src/main/resources/META-INF/dataflow-server-defaults.yml

```diff
@@ -41,7 +48,9 @@ spring:
     initialize-schema: never
   jpa:
     properties:
-      hibernate.id.new_generator_mappings: true
+      hibernate:
+        id.new_generator_mappings: true
+        generate_statistics: true
```

Comment:
While this is good for metrics generation, do you think this will have cause any performance hit (given we set it here by default)?

Reply:
You have a point
Not sure how significant, but the generate_statistics would defiantly have an impact on the performance.
We can move the configuration out from the defaults and document how to enable it when needed. If not enabled the dashboard charts will be empty. 
Same applies for the Skipper's configuration as well.

Reply:
I've updated this and the skipper PR to remove the generate_staistics=true from the default configuration.

Reply:
Maybe you can add the documentation part to enable this as part of this PR as well?

Reply:
This will be a separate issue: https://github.com/spring-cloud/spring-cloud-dataflow/issues/4102